CHANGELOG.md

```diff
@@ -78,6 +110,10 @@
 
 [Full Changelog](https://github.com/parallaxsecond/parsec-client-rust/compare/0.5.0...0.6.0)
 
+**Merged pull requests:**
+
+- Added asymmetric encrypt and decrypt [\#36](https://github.com/parallaxsecond/parsec-client-rust/pull/36) ([sbailey-arm](https://github.com/sbailey-arm))
+
```

Comment:
This is confusing. I wonder why it keeps reordering them like that :| I think last time I checked it seemed to be because of the order in which it was receiving some responses from Github....

Reply:
as long as the contents stay the same 🤡

Reply:
well, they clearly don't, since some stuff was moved from being associated with one version to being associated with a different one

Reply:
re-running the tool again gives me different results everytime

Reply:
yeah, I know, that's what I was saying - maybe keep the latest tag and copy it on top of the old file.

Reply:
D 🔛 E